tests/api/test_topic_entity_tag.py

```diff
@@ -172,7 +173,11 @@ def test_get_all_reference_tags(self, auth_headers): # noqa
 
     @pytest.mark.webtest
```

Comment:
@valearna should this still be a webtest, given it mocks ?

Reply:
Good catch @azurebrd ! I'll create a new PR for this. There is also another test with the same issue.

Reply:
Cool, thanks !  Missed that one :)